lib/AtomSpaceConfig.cmake.in

```diff
@@ -2,13 +2,63 @@
 
 include("@CMAKE_INSTALL_PREFIX@/lib/cmake/AtomSpace/AtomSpaceTargets.cmake")
 
+# these names are used in some projects e.g. opencog
+set(ATOMSPACE_atomproto_LIBRARY atomproto)
+set(ATOMSPACE_atombase_LIBRARY atombase)
+set(ATOMSPACE_atomcore_LIBRARY atomcore)
+set(ATOMSPACE_attentionbank_LIBRARY attentionbank)
+set(ATOMSPACE_atomspaceutils_LIBRARY atomspaceutils)
+set(ATOMSPACE_atomspace_LIBRARY atomspace)
+set(ATOMSPACE_truthvalue_LIBRARY truthvalue)
+set(ATOMSPACE_atomutils_LIBRARY atomutils)
+set(ATOMSPACE_clearbox_LIBRARY clearbox)
+set(ATOMSPACE_lambda_LIBRARY lambda)
+set(ATOMSPACE_persist_LIBRARY persist)
+set(ATOMSPACE_query_LIBRARY query)
+set(ATOMSPACE_execution_LIBRARY execution)
+set(ATOMSPACE_ruleengine_LIBRARY ruleengine)
+set(ATOMSPACE_smob_LIBRARY smob)
+set(ATOMSPACE_unify_LIBRARY unify)
```

Comment:
Following libraries are absent:
ATOMSPACE_LIBRARY - required by opencog/opencog
ATOMSPACE_PythonEval_LIBRARY - requried by opencog/benchmark
ATOMSPACE_persist-zmq_LIBRARY

Reply:
It seems there is no persist-zmq library, where it is defined? opencog/persist/zmq/atomspace/CMakeLists.txt defines only  zmqatoms..

Reply:
Yes, I have double checked, don't see it as well.